### PR TITLE
Add proposed RFC template and description

### PR DESCRIPTION
**Pull request by CjS77**
This is a suggested template and process management outline for RFC documents. It is loosely based on the [Consensus-Oriented Specification System](https://rfc.unprotocols.org/spec:2/COSS/)

### CODE REVIEW

File: src/RFC_template.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Long title
+
+![status: raw](theme/images/status-raw.svg)
+
+**Status**: raw
+
+**Editor**: <Editor Name> <email>
+
+**Contributors**: <List of contributors>
```
Comment (hansieodendaal) [inline src/RFC_template.md]: Combine 'Editor' and 'Contributors' into 'Maintainers'

File: src/RFC_template.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+# License
+
+[ The 3-Clause BSD License](https://opensource.org/licenses/BSD-3-Clause). Copyright 2018. <COPYRIGHT HOLDER>
```
Comment (hansieodendaal) [inline src/RFC_template.md]: I think 'The 3-Clause BSD License' should be on its own line

Comment (hansieodendaal) [inline src/RFC_template.md]: The full stop after 'Copyright 2018.' should be removed

Comment (hansieodendaal) [inline src/RFC_template.md]: Replace '2018' with '\<YEAR\>'

Comment (hansieodendaal) [inline src/RFC_template.md]: Should '\<COPYRIGHT HOLDER\>' not be replaced by 'Tari Project' or something?

File: src/RFC_template.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
+THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and
+"OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+## Goals
+
```
Comment (hansieodendaal) [inline src/RFC_template.md]: Add '<Goals>' under heading to be in line with the other place holders

File: src/RFC_template.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+## Description
+
```
Comment (hansieodendaal) [inline src/RFC_template.md]: Add '<Description>' under heading to be in line with the other place holders

File: src/RFC_template.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
+# [RFC-Number]-Short Title
```
Comment (hansieodendaal) [inline src/RFC_template.md]: Some guidance on number format perhaps?

Comment (hansieodendaal) [inline src/RFC_template.md]: Use of place holders, i.e. '[\<RFC-Number\>]-\<Short Title\>

File: src/RFC_template.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+## Long title
+
+![status: raw](theme/images/status-raw.svg)
+
+**Status**: raw
```
Comment (hansieodendaal) [inline src/RFC_template.md]: '**Status**: raw' could be removed in lieu of the image that says the same thing

File: src/about.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
+# The Tari RFCs
+
+Tari is a community-driven project. The documents presented in this RFC collection have typically gone through several
+iterations before reaching this point:
```
Comment (hansieodendaal) [inline src/about.md]: This '_The documents presented in this RFC collection have typically gone through several iterations before reaching this point:_' is in contrast to the 'Raw' status, which says '_Lots of iteration expected._'
The statement could just be removed.

File: src/about.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+* Ideas and questions are posted in #tari-dev on [#FreeNode IRC](https://freenode.net/). This is typically short-form
+  content with rapid feedback. Often, these conversations will lead to someone posting a proposal as a [pull request] into
+  [proposals].
+* [Proposals] get debated and refined using Github's Review and [pull request] systems. After the content has
+  stabilised, and the community accepts the proposal, it can be written up as a formal RFC.
```
Comment (hansieodendaal) [inline src/about.md]: Amerivan spelling '_stabilized_' vs '_stabilised_'?

File: src/about.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+New RFC's should follow the format given in the [RFC template](./RFC_template.md).
+## Lifecycle
+
+RFCs go through the following lifecycle, which roughly corresponds to the [COSS](https://rfc.unprotocols.org/spec:2/COSS/):
+
+| Status     | | Description                                              |
+|:-------    |:---|:---------------------------------------------------------|
+| Raw        |![raw](theme/images/status-raw.svg)| Newly submitted. Lots of iteration expected.             |
+| Draft      |![draft](theme/images/status-draft.svg)| Near final form. Final additions and changes still accepted |
+| Stable     |![stable](theme/images/status-stable.svg)| Typographical and cosmetic changes excepted, no further changes will be accepted. Changes to the Tari wrt a stable RFC will lead to the RFC becoming Deprecated, or Retired |
```
Comment (hansieodendaal) [inline src/about.md]: '_Changes to the Tari **code base** wrt a stable RFC will lead to the RFC becoming **Dated,** Deprecated or Retired_'

File: src/about.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+* RFCs are "Requests for Comment", so although the proposals in these documents are usually well-thought out, they are
+  not cast in stone. RFCs can, and should, undergo further evaluation and discussion by the community. RFC comments are
+  best made using Github [issues].
+
+New RFC's should follow the format given in the [RFC template](./RFC_template.md).
+## Lifecycle
+
+RFCs go through the following lifecycle, which roughly corresponds to the [COSS](https://rfc.unprotocols.org/spec:2/COSS/):
+
+| Status     | | Description                                              |
```
Comment (hansieodendaal) [inline src/about.md]: Missing full stops in '_Description_' column

File: src/about.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+|:-------    |:---|:---------------------------------------------------------|
+| Raw        |![raw](theme/images/status-raw.svg)| Newly submitted. Lots of iteration expected.             |
+| Draft      |![draft](theme/images/status-draft.svg)| Near final form. Final additions and changes still accepted |
+| Stable     |![stable](theme/images/status-stable.svg)| Typographical and cosmetic changes excepted, no further changes will be accepted. Changes to the Tari wrt a stable RFC will lead to the RFC becoming Deprecated, or Retired |
+| Deprecated |![deprecated](theme/images/status-deprecated.svg)| This RFC has been replaced by a newer RFC document, but is still is use in Tari code |
```
Comment (hansieodendaal) [inline src/about.md]: Add row for status 'Out of Date', or maybe 'Dated' to be more concise, SVG image (maybe dark green), as well as corresponding image and text, maybe 'The source of truth has changed, but this is still the best RFC we have.'

File: src/RFC_template.md (hunk 2 above)
Comment (CjS77) [reply src/RFC_template.md]: COSS says to not take copyright from individuals. I'll add as a default.

Comment (CjS77) [reply src/RFC_template.md]: Done

Comment (CjS77) [reply src/RFC_template.md]: Done

Comment (CjS77) [reply src/RFC_template.md]: Done

File: src/RFC_template.md (hunk 1 above)
Comment (CjS77) [reply src/RFC_template.md]: Good suggestion

File: src/about.md (hunk 7 above)
Comment (CjS77) [reply src/about.md]: It's not wrong though. Maybe we go straight to draft status.

File: src/about.md (hunk 9 above)
Comment (CjS77) [reply src/about.md]: done

File: src/about.md (hunk 10 above)
Comment (CjS77) [reply src/about.md]: Done

File: src/about.md (hunk 11 above)
Comment (CjS77) [reply src/about.md]: Cool, but I'll go with Tari purple :)

File: src/RFC_template.md (hunk 6 above)
Comment (CjS77) [reply src/RFC_template.md]: COSS has both and I also wondered why the duplication. Presumably they had a reason for the redundancy (maybe some browsers  don't render it / the link is lost?)

I'm willing to drop it if folks can't come up with a reason to keep it

File: src/RFC_template.md (hunk 3 above)
Comment (CjS77) [reply src/RFC_template.md]: Sorry, I don't understand your comment

File: src/RFC_template.md (hunk 5 above)
Comment (CjS77) [reply src/RFC_template.md]: <> Are pretty clearly placeholders imo

Comment (CjS77) [reply src/RFC_template.md]: Ok, see what you're getting at. I'll change them

**Pull request merged**